warehouse/scripts/publish.py

```diff
@@ -477,14 +477,19 @@ def _publish_exposure(
 
                     if publish:
                         typer.secho(publish_msg, fg=typer.colors.GREEN)
-                        with open(fpath, "rb") as fp:
-                            upload_to_ckan(
-                                url=destination.url,
-                                fname=fname,
-                                fsize=fsize,
-                                file=fp,
-                                resource_id=resource.id,
-                            )
+                        try:
+                            with open(fpath, "rb") as fp:
+                                upload_to_ckan(
+                                    url=destination.url,
+                                    fname=fname,
+                                    fsize=fsize,
+                                    file=fp,
+                                    resource_id=resource.id,
+                                )
+                        except requests.exceptions.RequestException as e:
+                            print(e)
+                            continue
```

Comment:
I don't think you need this statement here. The end of the loop iteration, a few lines further down, manually deletes the `df` dataframe after the if-else logic but before the loop iteration completes, which was probably done for resource constraint reasons. We likely want to still be able to reach that line after an error is logged, after which point the loop will still move on to its next iteration as normal.

Reply:
makes a lot of sense, I went back on forth on this for a long time too hah